examples/Reading in linearised models.ipynb

```diff
@@ -0,0 +1,376 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.        plot_dfs(dfs, sns.lineplot, legend=[i.base_name for i in mod.shocks]);
</pre><p>There's a plotting function for IRFs in <code>gEconpy.plotting.plot_irf</code>,was it not suitable for your needs? I should tune it up if so, so you don't need to write any custom plotting functions in a notebook like this.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/jessegrabowski/gEconpy/pull/33/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>